root-sync/base/crossplane/project/kustomization.yaml

```diff
@@ -2,6 +2,6 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
+  # - composition-project.yaml 
   - kubernetes.yaml
```

Comment:
Kustomize is not building. It should be one level out in `root-sync/base/crossplane/kustomization.yaml`? It might be useful to add a workflow that runs `kustomize build` to ensure our manifests are built properly.

Reply:
Thanks! We've added that in #113.